db/dbqueue.c

```diff
@@ -1197,7 +1194,7 @@ static void dbqueue_flush(struct dbtable *db, int consumern)
     logmsg(LOGMSG_INFO, "Beginning flush for queue '%s' consumer %d\n",
            db->tablename, consumern);
 
-    if (db->dbenv->master != machine()) {
+    if (db->dbenv->master != gbl_mynode) {
```

Comment:
Check if we need strcmp instead

Reply:
Old machine() simply returned gbl_mynode.
```
char *machine(void) { return gbl_mynode; }
```